content/realtime/inband-occupancy.textile

```diff
@@ -0,0 +1,124 @@
+* @metrics.<category>@:  this enables events whose @data@ payload contains an @Occupancy@ value containing the occupancy, that is the client count, for only the given category. Events are sent when the count for the given category changes, up to a maximum rate of one message per second.
+
+Channel parameters are a general mechanism for specifying attributes of a channel attachment. See the "Channel parameters documentation":/realtime/channel-params for further details.
+
+The following example shows how to specify the occupancy channel parameter:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+In this case all occupancy metrics are contained in occupancy messages.
+
+The following example shows how to specify a category:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics.subscribers' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+In this case only occupancy metrics for @subscribers@ are contained in occupancy messages.
+
+h2(#occupancy-events). Occupancy events 
+
+Events in a channel that contain metadata have an event name qualified with the @[meta]prefix@. For example, occupancy messages have an event name of @[meta]occupancy@. Messages published by clients do not contain this metadata. The following code snippet illustrates this:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+
+channel.subscribe('[meta]occupancy', (message) => {
+  console.log('inband: ', message.data);
+});
+```
+
+In this example only occupancy messages will be logged to the console, not user-specified messages.
+
+An example inband channel occupancy message is shown here:
+
+```[json]
+{
+  name: '[meta]occupancy',
+  id: 'V12G5ABc_M:0:0',
+  timestamp: 1612286351217,
+  clientId: undefined,
+  connectionId: undefined,
+  connectionKey: undefined,
+  data: {
+    metrics: {
+      connections: 1,
+      publishers: 1,
+      subscribers: 1,
+      presenceConnections: 1,
+      presenceMembers: 0,
+      presenceSubscribers: 1
+    }
+  },
+  encoding: null,
+  extras: undefined,
+  size: undefined
+}
+```
+
+Occupancy events have a payload in their @data@ attribute which is an @Occupancy@ value. An example of the occupancy event data is shown here:
+
+```[json]
+{
+  metrics: {
+    connections: 4,
+    publishers: 4,
+    subscribers: 4,
+    presenceConnections: 4,
+    presenceMembers: 2,
+    presenceSubscribers: 4
+  }
+}
+```
+
+If subscribing to only a single category, then only that member is present, as shown in the following example for the @subscribers@ category: 
+
+```[javascript]
+{
+  name: '[meta]occupancy',
+  data: {
+    metrics: {
+      subscribers: 2
+    }
+  }
+}
+```
+
+h2(#see-also). See also
```

Comment:
Nothing in this document about the permissions you need to be able to do this? https://gist.github.com/paddybyers/d350f491d0807360620e37eb4d3af66c#gistcomment-3245519

Reply:
Added dedicated section on permissions.